### ISSUE TITLE
Uncaught Error: Failed to load gRPC binary module

### ISSUE THREAD

**Issue by kroggen** [aergoio/athena-ide-atom#7]
[Enter steps to reproduce:]

1. Open athena-ide-atom
2. Click on Refresh button

**Atom**: 1.40.1 x64
**Electron**: 3.1.10
**OS**: Mac OS X 10.13.6
**Thrown From**: [athena-ide-atom](https://github.com/aergoio/athena-ide-atom) package 0.4.2

### Stack Trace

Uncaught Error: Failed to load gRPC binary module because it was not installed for the current system
Expected directory: electron-v3.1-darwin-x64-unknown
Found: [electron-v2.0-darwin-x64-unknown]
This problem can often be fixed by running "npm rebuild" on the current system
Original error: Cannot find module '/Users/bernardo/.atom/packages/athena-ide-atom/node_modules/grpc/src/node/extension_binary/electron-v3.1-darwin-x64-unknown/grpc_node.node'

```
At /Users/bernardo/.atom/packages/athena-ide-atom/node_modules/react-dom/cjs/react-dom.production.min.js:17

Error: Failed to load gRPC binary module because it was not installed for the current system
Expected directory: electron-v3.1-darwin-x64-unknown
Found: [electron-v2.0-darwin-x64-unknown]
This problem can often be fixed by running "npm rebuild" on the current system
Original error: Cannot find module '/Users/bernardo/.atom/packages/athena-ide-atom/node_modules/grpc/src/node/extension_binary/electron-v3.1-darwin-x64-unknown/grpc_node.node'
    at /packages/athena-ide-atom/node_modules/grpc/src/grpc_extension.js:53:17)
    at /packages/athena-ide-atom/node_modules/grpc/src/grpc_extension.js:64:3)
    at Module.get_Module._compile (/Applications/Atom.app/Contents/Resources/app/static/<embedded>:11:147429)
    at Object.value [as .js] (/Applications/Atom.app/Contents/Resources/app/static/<embedded>:11:150977)
    at Module.load (internal/modules/cjs/loader.js:620:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:559:12)
    at Function.Module._load (internal/modules/cjs/loader.js:551:3)
    at Module.require (/app.asar/static/index.js:72:46)
    at require (/Applications/Atom.app/Contents/Resources/app/static/<embedded>:11:146745)
    at /packages/athena-ide-atom/node_modules/grpc/src/client_interceptors.js:144:12)
    at /packages/athena-ide-atom/node_modules/grpc/src/client_interceptors.js:1446:3)
    at Module.get_Module._compile (/Applications/Atom.app/Contents/Resources/app/static/<embedded>:11:147429)
    at Object.value [as .js] (/Applications/Atom.app/Contents/Resources/app/static/<embedded>:11:150977)
    at Module.load (internal/modules/cjs/loader.js:620:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:559:12)
    at Function.Module._load (internal/modules/cjs/loader.js:551:3)
    at Module.require (/app.asar/static/index.js:72:46)
    at require (/Applications/Atom.app/Contents/Resources/app/static/<embedded>:11:14

**Comment by kroggen** [comment]
This happened after the Atom was updated.

As a workaround I uninstalled the last version and installed version 1.38 again.

**Comment by acktsap** [comment]
Thanks and Sorry for late :)

It happened because the Atom upgraded electron version.
It's been fixed in 9965b98bbc144de8e06fe4f310ad495c390c2943
It will be available soon in v0.4.3
